### PR TITLE
enhancement Add types to app folder as well

### PR DESCRIPTION
**Pull request by snewcomer**
I think this will work so you don't have to reach into `addon/types` to import. ember-cli will look for it in the `app/` namespace I believe.  @josemarluedke Lmk if you have the same thought as well.

ref #424

### CODE REVIEW

**Comment by snewcomer** [comment]
Or since this is at build time, you would reference `ember-changeset/app/types`?

**Comment by josemarluedke** [comment]
@snewcomer Yeah, in the app folder would not work either. I believe it needs to be on the root folder. eg. `ember-changeset/types/index.d.ts`

**Comment by snewcomer** [comment]
That makes a lot more sense :)

**Comment by josemarluedke** [comment]
I just tested here and moving to root does work correctly.

File: types/index.d.ts [hunk 1]
```diff
@@ -0,0 +1,20 @@
+import {
```
Comment (josemarluedke) [inline types/index.d.ts]: Should we also remove this file from `addon/types/index.d.ts`?

**Pull request merged**